force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -65,17 +107,34 @@ public with sharing class DummyJSONCallout {
      */
 	@TestVisible // Allows test class to see this method. Since it is private, it would not be visible otherwise.
 	private static Contact createContactFromJson(String jsonResponse) {
```

Comment:
It looks like this is duplicated code from the HTTP method above so you can probably refactor your callout method to use this method for creating the contact 🙂